atom/browser/ui/tray_icon_cocoa.mm

```diff
@@ -254,9 +254,23 @@ - (void)rightMouseUp:(NSEvent*)event {
 }
 
 - (NSDragOperation)draggingEntered:(id <NSDraggingInfo>)sender {
+  trayIcon_->NotifyDragEntered();
   return NSDragOperationCopy;
 }
 
+- (void)draggingExited:(id <NSDraggingInfo>)sender {
+  trayIcon_->NotifyDragExited();
+}
+
+- (void)draggingEnded:(id <NSDraggingInfo>)sender {
+  trayIcon_->NotifyDragEnded();
+}
+
+- (BOOL)prepareForDragOperation:(id <NSDraggingInfo>)sender {
+  trayIcon_->NotifyDrop();
```

Comment:
@zcbenz Thoughts on whether the drop event notification occurs at the correct place?